build-system/babel-plugins/babel-plugin-transform-annotation-to-extern/index.js

```diff
@@ -93,12 +95,13 @@ module.exports = function(babel) {
 
         if (!TYPEDEFS.has(typedefName)) {
```

Comment:
Isn't this an error case?

Reply:
@jridgewell do you mean where theres a duplicate typedef name?

Reply:
Yes

Reply:
@jridgewell it should but the linter should catch it from a project level, and in program level it should be an error too.

I'm probably clearing the Map in the wrong place, im currently doing it in `pre` (is that executed only once?), should i be doing it on Program enter?

Reply:
I would throw here.

Pre is executed once per plugin. `Program:enter` is executed once per file.

Reply:
done.